src/main/java/com/rev_connect_api/models/BusinessProfile.java

```diff
@@ -0,0 +1,111 @@
+package com.rev_connect_api.models;
+
+import jakarta.persistence.*;
+
```

Comment:
```suggestion
@Data
```
You can use this lombok annotation to create the getters/setters/hashcode/tostring methods for you